ash/src/entry.rs

```diff
@@ -29,40 +29,10 @@ pub struct Entry {
 /// Vulkan core 1.0
 #[allow(non_camel_case_types)]
 impl Entry {
-    /// Load entry points from a Vulkan loader linked at compile time
-    ///
-    /// Note that instance/device functions are still fetched via `vkGetInstanceProcAddr` and
-    /// `vkGetDeviceProcAddr` for maximum performance.
-    ///
-    /// ```no_run
-    /// use ash::{vk, Entry};
-    /// # fn main() -> Result<(), Box<dyn std::error::Error>> {
-    /// let entry = Entry::new();
-    /// let app_info = vk::ApplicationInfo {
-    ///     api_version: vk::make_api_version(0, 1, 0, 0),
-    ///     ..Default::default()
-    /// };
-    /// let create_info = vk::InstanceCreateInfo {
-    ///     p_application_info: &app_info,
-    ///     ..Default::default()
-    /// };
-    /// let instance = unsafe { entry.create_instance(&create_info, None)? };
-    /// # Ok(()) }
-    /// ```
-    #[cfg(feature = "linked")]
-    #[cfg_attr(docsrs, doc(cfg(feature = "linked")))]
-    pub fn new() -> Self {
-        // Sound because we're linking to Vulkan, which provides a vkGetInstanceProcAddr that has
-        // defined behavior in this use.
-        unsafe {
-            Self::from_static_fn(vk::StaticFn {
-                get_instance_proc_addr: vkGetInstanceProcAddr,
-            })
-        }
-    }
-
     /// Load default Vulkan library for the current platform
     ///
+    /// Prefer this over [`linked`](Self::linked) when Vulkan support is not required at runtime.
```

Comment:
Vulkan support is always required... _linking Vulkan at runtime_ however, is not :)

You may want to mention though that using `linked` needs the aptly named feature to be enabled.

Reply:
That wording was intended; tweaked it to be clearer. The `#[cfg_attr(docsrs, doc(cfg(feature = "loaded")))]` should take care of the latter part.

Reply:
Much better, thanks!